packages/api-sdk/src/medical/models/patientDTO.ts

```diff
@@ -0,0 +1,43 @@
+export type PatientDTO = {
+  id: string;
+  eTag?: string | undefined;
+  firstName: string;
+  lastName: string;
+  dob: string;
+  genderAtBirth: string;
+  personalIdentifiers?: PersonalIdentifier | undefined;
+  facilityIds: string[];
+  address: Address | Address[];
+  contact?: Contact | Contact[] | undefined;
+};
+
+type Address = {
+  addressLine1?: string | undefined;
```

Comment:
nit: when we indicate the prop is optional w/ `?` we don't need to add `| undefined`